src/components/icon/icons/Play.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { InternalIconProps } from '../Icon.types'
+import { colors } from '../../../theme/colors'
+
+const Play = ({ colors: { primary }, ...props }: InternalIconProps) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    fill="none"
+    viewBox="0 0 32 32"
+    {...props}
+  >
+    <path
+      fill={primary}
+      fillRule="evenodd"
+      d="M4 16C4 9.4 9.4 4 16 4s12 5.4 12 12-5.4 12-12 12S4 22.6 4 16zM16 6C10.5 6 6 10.5 6 16s4.5 10 10 10 10-4.5 10-10S21.5 6 16 6z"
+      clipRule="evenodd"
+    />
+    <path
+      fill={colors.eucalyptusGreen}
```

Comment:
This should be the `secondary` colour 🙂 this can be destructured from the `colors` prop which is coming into the component (similar to `primary`).

Reply:
I used the eucalyptusGreen because in the design the centre of the play button is green, and when I use secondary it is indigo. I changed it to use secondary now, let me know what you prefer! 😃

Reply:
@SofiaFredriksson apologies, my bad. This **will** be the correct colour once the PR https://github.com/PrideInLondon/pride-london-web/pull/1475 is merged. I'll get on that now, I thought it had already gone through!

Reply:
@SofiaFredriksson PR #1475 is merged :)

Reply:
@SofiaFredriksson if you merge in the latest `master` to your branch, you should now have this update! 😄